### ISSUE TITLE
Black output, learning_rate=0, no output.

### ISSUE THREAD

**Issue by hocuf** [milesial/Pytorch-UNet#499]
We performed a segmentation study on our own dataset using the UNet model, but we did not achieve the desired success. We tried various scenarios with different pixel sizes, image and mask formats and various pre-processing techniques. We failed in every scenario. The output is always black. Learning rate is always zero.

The scenarios we tried are as follows: Hyperparameters changes, Normalization (Min-Max), Normalization (Z-Score), Learning Rate change, Adding Loss Function, Creating Dual Classes, Batch Normalization, Img_Scale application and Optimizer changes (Adam, ReLU). For each scenario, we experimented with 256x256, 512x512, 2180x2180 and 2112x2112 pixel sizes in PNG format.

As a result of all these experiments, our model was not successful and the desired accuracy was not achieved.
